src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1353,6 +1353,18 @@
   <data name="PreferIsEmptyOverCountTitle" xml:space="preserve">
     <value>Prefer IsEmpty over Count</value>
   </data>
+  <data name="UseEnvironmentProcessIdDescription" xml:space="preserve">
+    <value>Environment.ProcessId is simpler and faster than Process.GetCurrentProcess().Id.</value>
+  </data>
+  <data name="UseEnvironmentProcessIdMessage" xml:space="preserve">
+    <value>Use Environment.ProcessId instead of Process.GetCurrentProcess().Id.</value>
```

Comment:
```suggestion
    <value>Use 'Environment.ProcessId' instead of 'Process.GetCurrentProcess().Id'</value>
```

Reply:
There are tons of ending periods in the resx.  If we really don't want them, is someone going to go through and clean them all up soon?
cc: @mavasani

Reply:
Yes, we need to write an analyzer/code fix to cleanup and strictly enforce any desired patterns. I'll try to complete the analyzer/fixer that @Evangelink started in this space.